# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: kov@debian.org-20120510213816-1l67h6vyp9npe24m
# target_branch: http://bazaar.launchpad.net/~kees/apparmor/debian/
# testament_sha1: 575cc0773e515fd50f78106a25444c3356b29ff3
# timestamp: 2012-05-10 18:38:55 -0300
# base_revision_id: kees@outflux.net-20120505224315-gxa5bxq6doy792ro
# 
# Begin patch
=== modified file 'utils/aa-genprof'
--- utils/aa-genprof	2012-01-13 00:55:46 +0000
+++ utils/aa-genprof	2012-05-10 21:38:16 +0000
@@ -67,8 +67,7 @@
 # tell 'em how to use it...
 &usage && exit if $help;
 
-my $sd_mountpoint = check_for_subdomain();
-unless ($sd_mountpoint) {
+unless (!system ("aa-status --enabled")) {
     fatal_error(gettext("AppArmor does not appear to be started. Please enable AppArmor and try again."));
 }
 

# Begin bundle
IyBCYXphYXIgcmV2aXNpb24gYnVuZGxlIHY0CiMKQlpoOTFBWSZTWQXkbGUAAY/fgAAQcmP/93MA
AgC////4UANO8AUZAA0SajQxMaJ4ghpoYQ0AAMAAAAANBoA0A0BgAAAABoNAGgGgICU9qjelPUMQ
BoBoDQAAJJEyBoAJqYnqaCniTyMo2p6nkgYP3PAWytkCIZXgBZjW4hgQjJ3eQgKtmMGfTkXhLpmb
EZH4qkKduv2VXPMxBGnola+BRmdXpEEJSF4RHIOprHlazyFXcvYUFSLtP11rqnp99onCqH+/SpSt
vdTxevNl1pN4Y9yRMl3HZnooIHLBiC7XT9B5koKZVk7pCs49FOvxIVupDnW57kiy7K2r84ibircP
jpZz0UFsL+5qLNGryw4Lq12z2FbU4QyDYUlwP3sJwrFAgGeBjSHoiadYTcidcrpozZvo1pHpWZer
S+aIUnsqE9Ybv00T8JsAjSy5qEA1ONJo2jC8r3zhcmOLhVsoWqmy1Q/rrvZSBoz0zkhRqeo6uMZ8
drhNXuz75MaZr5XiKgipQVVUu9WEixlMRiLGhQV0RSxzoHWcjOm9Stka8JrZWXNONNgsFboOIqNk
KLDgsZuipGUcJd4nyQG0dtkiTnIKIxg8xKsTgUAepENQTLAIBSZ4890X/+luuWUoJmb9uvRxufaw
U7zHUnBn+Fnksr2+6ZLCyzyDn5x+qHpQhjjUL0U53+OOrjo9d7GCVQV+C5B2UnE6lu02fl57Pk34
stRcreLi8nxVSeRWKfjFXB4FN6KFofJ3rnupNy0OgmOYXC8k11ZX21aAhOYCzURWJ2mJqZ1GvAqY
87DriQFWsCSnF5GU63VzRUcYCoaoSE9ABHAUNV292g46zJjkWlXxzdJXOPAuF3Ib5OdTLpuJuIUV
/di6nleSVzmr+020dzQHCy7Ql8a3vwCPaFb9453pnK8Pc9hnnHUKQ5CqeHE2ZlKe/1gE+qq23K2O
jclW66ofnGI+BCZ6TKdavmCb1TnVWhD1FQBFkyt5yS2Jg8c5i8iFqVC8Xq6/1XYl0VPkKiOjMnTF
gqX/WekFPn6JqIs5MHOeZEeciKU/Ryrnx/StUWnngmXXI6obrucp4u+4XrslSDUHBuFKiOnR94Wx
XDDtg/YjgnNLUwgqqM95gtJ6QrB6g5ZUHplvFcafzeW9zL0W2Selcsgf7iyzVah2JlcqV57iqv01
KJfwXckU4UJAF5GxlA==
